pom.xml

```diff
@@ -115,7 +115,7 @@
         <maven.compiler.target>1.8</maven.compiler.target>
         <version.spring.boot>2.7.6</version.spring.boot>
         <version.guava>31.1-jre</version.guava>
-        <version.spring.cloud>2021.0.5</version.spring.cloud>
+        <version.spring.cloud>2022.0.0</version.spring.cloud>
```

Comment:
For spring-cloud-azure 4.x, the compatible spring-cloud version is still 2020.0.x and 2021.0.y.
Refs: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/spring/CHANGELOG.md#450-2022-12-06